pkg/framework/framework.go

```diff
@@ -52,6 +52,18 @@ func NewFramework() (*Framework, error) {
 		return nil, err
 	}
 
+	// Initialize Release Controller
```

Comment:
These controllers are just commented out below. Please either uncomment them, or delete them from the commented out section.